main.py

```diff
@@ -1,10 +1,34 @@
 import init_django_orm  # noqa: F401
+import json
+
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+    for player in players:
```

Comment:
```suggestion
    for player_name, player_data in players.items():
```
This will allow you to simplify the code a bit))

Reply:
I don`t need items because i use only keys in the dist. And as we know by default in dict user function keys()

Reply:
This will help simplify your code. Instead of always using `players[player]`, you can use `player_data`, which is a better and more efficient approach.